tools/lib/gitlint-rules.py

```diff
@@ -10,87 +10,258 @@
-    'uses', 'using', 'used',
+    "adds",
+    "adding",
+    "added",
+    "allows",
+    "allowing",
+    "allowed",
+    "amends",
+    "amending",
+    "amended",
+    "bumps",
+    "bumping",
+    "bumped",
+    "calculates",
+    "calculating",
+    "calculated",
+    "changes",
+    "changing",
+    "changed",
+    "cleans",
+    "cleaning",
+    "cleaned",
+    "commits",
+    "committing",
+    "committed",
+    "corrects",
+    "correcting",
+    "corrected",
+    "creates",
+    "creating",
+    "created",
+    "darkens",
+    "darkening",
+    "darkened",
+    "disables",
+    "disabling",
+    "disabled",
+    "displays",
+    "displaying",
+    "displayed",
+    "documents",
+    "documenting",
+    "documented",
+    "drys",
+    "drying",
+    "dryed",
+    "ends",
+    "ending",
+    "ended",
+    "enforces",
+    "enforcing",
+    "enforced",
+    "enqueues",
+    "enqueuing",
+    "enqueued",
+    "extracts",
+    "extracting",
+    "extracted",
+    "finishes",
+    "finishing",
+    "finished",
+    "fixes",
+    "fixing",
+    "fixed",
+    "formats",
+    "formatting",
+    "formatted",
+    "guards",
+    "guarding",
+    "guarded",
+    "handles",
+    "handling",
+    "handled",
+    "hides",
+    "hiding",
+    "hid",
+    "increases",
+    "increasing",
+    "increased",
+    "ignores",
+    "ignoring",
+    "ignored",
+    "implements",
+    "implementing",
+    "implemented",
+    "improves",
+    "improving",
+    "improved",
+    "keeps",
+    "keeping",
+    "kept",
+    "kills",
+    "killing",
+    "killed",
+    "makes",
+    "making",
+    "made",
+    "merges",
+    "merging",
+    "merged",
+    "moves",
+    "moving",
+    "moved",
+    "permits",
+    "permitting",
+    "permitted",
+    "prevents",
+    "preventing",
+    "prevented",
+    "pushes",
+    "pushing",
+    "pushed",
+    "rebases",
+    "rebasing",
+    "rebased",
+    "refactors",
+    "refactoring",
+    "refactored",
+    "removes",
+    "removing",
+    "removed",
+    "renames",
+    "renaming",
+    "renamed",
+    "reorders",
+    "reordering",
+    "reordered",
+    "replaces",
+    "replacing",
+    "replaced",
+    "requires",
+    "requiring",
+    "required",
+    "restores",
+    "restoring",
+    "restored",
+    "sends",
+    "sending",
+    "sent",
+    "sets",
+    "setting",
+    "separates",
+    "separating",
+    "separated",
+    "shows",
+    "showing",
+    "showed",
+    "simplifies",
+    "simplifying",
+    "simplified",
+    "skips",
+    "skipping",
+    "skipped",
+    "sorts",
+    "sorting",
+    "speeds",
+    "speeding",
+    "sped",
+    "starts",
+    "starting",
+    "started",
+    "supports",
+    "supporting",
+    "supported",
+    "takes",
+    "taking",
+    "took",
+    "testing",
+    "tested",  # 'tests' excluded to reduce false negative
+    "truncates",
+    "truncating",
+    "truncated",
+    "updates",
+    "updating",
+    "updated",
+    "uses",
+    "using",
+    "used",
```

Comment:
Probably want to exclude this file since this part at least had an intentional structure.